swarm/api/routes/wisdom.py

```
@@ -422,6 +425,18 @@ async def get_wisdom_artifacts(run_id: str):
     """
+    try:
+        validate_path_component(run_id, "run_id")
+    except ValueError as e:
+        raise HTTPException(
+            status_code=400,
+            detail={
+                "error": "invalid_request",
+                "message": str(e),
+                "details": {"run_id": run_id},
+            },
+        )
```

Ruff (0.15.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)